spring-boot/proxy/src/main/java/eu/coatrack/proxy/filters/post/ErrorLoggingFilter.java

```diff
@@ -83,11 +84,11 @@ public Object run() {
             log.warn("Response body and data stream are null - assuming that service is unavailable (logging status 503)");
 
             String apiKeyValue = SecurityContextHolder.getContext().getAuthentication().getCredentials().toString();
-            metricsCounterService.increment(
+            metricsCounterService.increment(new MetricsHolder(
```

Comment:
This additional wrapper objects seems to make the code more complex, to be discussed.